docs/source/roadmap.rst

```diff
@@ -1,67 +1,52 @@
-* **At-scale refinements:** Ensure event handling reliability, event storm resilience/throttling. Complete support for multi-node deployment of sensor containers and rule engines for resilience and throughput.
-* **Content management, revised for multi-node deployments:** provide platform support for content deployment to worker nodes with better integration with git/GitHub, to simplify development and deployment of "automation as code" at scale.
-* **Security hardening:** Complete security audit and address issues discovered so far.
-* **History and Audit service:** History view with advanced search over years worth of execution records, over multiple versions of continuously upgraded StackStorm.
-* **Multi target configurations for integration packs:** For a given integration pack, define and manage multiple targets - sets of configurations, so that the user can choose which one to use for a given action.
-* **First class Windows support:** switch to pywinrm for better license. Remote PowerShell via Powershell.REST.API. Windows-native ActionRunners. Windows supported st2workroom.
-* **Projects and Uber-flow:** introduce projects to group and manage rules and workflows. Handle versions and dependencies. "Productize" flow-rule-flow-rule chain pattern, aka "uber-flow". Manage large number of automations across users and teams, on a single StackStorm deployment at enterprise scale.
-* **RBAC refinements and improvements:**
-
-  * tag and property based filters, more refined and convenient access control
-  * permissions on key value objects, arbitrary triggers, support for a default role with is assigned to new users.
-  * WebUI for RBAC configuration
-  * RBAC for ChatOps - allow user to authenticate with StackStorm via bot on chat and when checking permissions directly check permissions of the user who triggered an action / ran a command. Allow introduce a special set of permission types for ChatOps.
-* **StackStorm Forge:** for Community and Enterprise integration and automation packs. Solve the problem of packs spread all over GitHub. Make integration and automation packs discoverable, continuously tested, and community rated
-* **More integration packs:** push more content to the community to help work with most common and widely used tool. Tell us if there is tool you love and think we should integrate with or better yet write a pack yourself.
-* **Action Output Structure Definition**: enable optional definition of action payload, so that it can be introspected and used when passing data between actions in workflows.
-* **Datastore:**  Hierarchical, pluggable and secure datastore to bring together all the gnarly config and key spread in operations environments.
-* **Documentation:** generate REST API docs.
-* **Flow v2:** Visualizing workflow executions. UI improvements, including more "operator" persona features.
+.. _platform:
 
+Core Platform
+-------------
```

Comment:
I'm -1 on introducing categories: many things will fall behind, and our community content is dynamic and situational. Let's stay with one list.